File: Ruby on Rails/questions/Q1_HelloWorld.rb
# Q1_HelloWorld.rb
# Create a simple Rails controller that displays "Hello, World!"

# config/routes.rb
Rails.application.routes.draw do
  get '/', to: 'hello#index'
end

# app/controllers/hello_controller.rb
class HelloController < ApplicationController
  def index
    @message = "Hello, World!"
  end
end

# app/views/hello/index.html.erb
<h1><%= @message %></h1>

File: Backend-Web/02-Web-Frameworks/DotNet/ASP.NET-Core/questions/Q1_HelloWorld.cs
// Q1_HelloWorld.cs
// Create a simple ASP.NET Core API endpoint that returns "Hello, World!"

using Microsoft.AspNetCore.Mvc;

[ApiController]
[Route("api/[controller]")]
public class HelloController : ControllerBase
{
    [HttpGet]
    public IActionResult Get()
    {
        return Ok(new { message = "Hello, World!" });
    }
}
